README.md

```diff
@@ -34,11 +38,11 @@ Here are all the supported values, including optional ones:
     {
       // Unique Open VSX extension ID in the form "<namespace>.<name>"
       "id": "rebornix.ruby",
-      // (OPTIONAL) The version to publish to Open VSX (defaults to the package.json version)
-      "version": "0.27.0",
       // Repository URL to clone and publish from
       "repository": "https://github.com/rubyide/vscode-ruby",
-      // (OPTIONAL) The Git branch, tag, or commit to check out before publishing (defaults to the repository's default branch)
+      // (RECOMMENDED) The version to publish to Open VSX (defaults to the package.json version)
+      "version": "0.27.0",
+      // (RECOMMENDED) The Git branch, tag, or commit to check out before publishing (defaults to the repository's default branch), should match "version"
```

Comment:
Due to the reason described above, it's ok to not match "version" (this could also be a "release" or "stale" branch, and you can even omit "version", which just serves to skip publishing when it's not necessary).

```suggestion
      // (RECOMMENDED) The Git branch, tag, or commit to check out before publishing (defaults to the repository's default branch)
```